drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -96,6 +94,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
       end
     end
 
+    def self.unit_types_for_project(project, available_only: false)
+      units = project.units
+      units = units.unoccupied if available_only
+
+      # Hash { unit type id => num unoccupied }
+      unit_type_to_availability = units.group(:unit_type_id).
+        pluck('unit_type_id, count(unit_type_id)').to_h
```

Comment:
Micro-optimization: I think it is equivalent (count by unit_type_id)
```suggestion
      unit_type_to_availability = units.group(:unit_type_id).count
```